src/ray/common/buffer.h

```diff
@@ -55,9 +55,9 @@ class LocalMemoryBuffer : public Buffer {
       : has_data_copy_(copy_data) {
     if (copy_data) {
       RAY_CHECK(data != nullptr);
-      buffer_.insert(buffer_.end(), data, data + size);
-      data_ = buffer_.data();
-      size_ = buffer_.size();
+      data_ = (uint8_t *)malloc(size);
```

Comment:
Use corresponding collection and release functions

malloc V.S. free,
new V.S. delete

Reply:
> Use corresponding collection and release functions
> 
> malloc V.S. free,
> new V.S. delete

fixed.